source/MRMesh/MRObjectGcode.cpp

```diff
@@ -9,11 +9,12 @@ namespace MR
 
 MR_ADD_CLASS_FACTORY( ObjectGcode )
 
-ObjectGcode::ObjectGcode()
+ObjectGcode::ObjectGcode( const CNCMachineSettings& cncSettings )
```

Comment:
please do not make it argument, just setup settings frim SceneSettings inside constructor like flatShading setting in ObjectMesh

Reply:
fixed